pom.xml

```diff
@@ -84,5 +84,16 @@
 			<version>4.12</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-simple</artifactId>
+            <version>1.7.36</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Why do you need this dependency? If you really need this, then it is better to use latest one - https://mvnrepository.com/artifact/org.slf4j/slf4j-simple/2.0.6

Also, it would be good if you format code indentations :)

Reply:
we dont use this anymore, I remove it